src/cryptsetup/cryptsetup-generator.c

```diff
@@ -367,12 +367,12 @@ static int create_disk(
 
         if (tmp)
                 fprintf(f,
-                        "ExecStartPost=/sbin/mke2fs '/dev/mapper/%s'\n",
+                        "ExecStartPost=" ROOTLIBEXECDIR "/systemd-makefs ext2 '/dev/mapper/%s'\n",
```

Comment:
Interesting! We talked about this for some time, I never suspected the change would be so tiny ;)